pkg/volume/emptydir/empty_dir_quota.go

```diff
@@ -69,10 +69,16 @@ func (edq *emptyDirQuotaMounter) SetUp(fsGroup *int64) error {
 
 func (edq *emptyDirQuotaMounter) SetUpAt(dir string, fsGroup *int64) error {
 	err := edq.wrapped.SetUpAt(dir, fsGroup)
-	if err == nil {
-		err = edq.quotaApplicator.Apply(dir, edq.medium, edq.pod, fsGroup, edq.quota)
+	if err != nil {
+		return err
 	}
-	return err
+
+	err = edq.quotaApplicator.Apply(dir, edq.medium, edq.pod, fsGroup, edq.quota)
+	if err != nil {
+		return err
+	}
+
+	return nil
```

Comment:
I aggre with you that just return "err" here to be a little more concise. But When I read code, I foud most code like below , Mabye we would like to add logs in the "if", or develop the fuction in future.
pkg/volume/emptydir/quota.go
	if fsGroup == nil {
		// This indicates the operation matched an SCC with FSGroup strategy RunAsAny.
		// Not an error condition.
		glog.V(5).Infof("Unable to apply XFS quota, no FSGroup specified.")
		return nil
	}

	volDevice, err := xqa.getFSDevice(dir)
	if err != nil {
		return err
	}

	err = xqa.applyQuota(volDevice, quota, *fsGroup)
	if err != nil {
		return err
	}

	return nil
}